contracts/interfaces/0.8.x/IFilteredMinterSEAV0.sol

```diff
@@ -86,15 +86,13 @@ interface IFilteredMinterSEAV0 is IFilteredMinterV2 {
 
     function resetAuctionDetails(uint256 _projectId) external;
 
-    function settleAndInitializeAuction(
+    function settleAndCreateBid(
         uint256 _settleTokenId,
         uint256 _initializeTokenId
```

Comment:
Should be named `_bidTokenId` I believe.

Reply:
good call, updated!